docs/docs/types/form.md

```diff
@@ -0,0 +1,48 @@
+Used primarily for data collection or user identification. Supporting a variety of form fields and options for passing data to various end-points this type presents a form for the user to fill out and submit.
+
+``` javascript
+var module = pathfora.Form({
+	// form module settings
+});
+
+pathfora.initializeWidgets([ module ]);
+```
+
+
+## layout
+
+Define which layout type the form module should use.
+
+| Value | Type | Behavior |
+|---|---|---|
+| modal | string | module uses a [modal layout](../layouts/modal.md) |
+| slideout | string | module uses a [slideout layout](../layouts/slideout.md) |  
+
+### Form - [Live Preview](../../examples/preview/types/form/modal.html)
+
+![Form Modal](../examples/img/types/form/modal.png)
+
+<pre data-src="../../examples/src/types/form/modal.js"></pre>
+
+
+### Slideout - [Live Preview](../../examples/preview/types/form/slideout.html)
+
+![Form Slideout](../examples/img/types/form/slideout.png)
+
+<pre data-src="../../examples/src/types/form/slideout.js"></pre>
+
+
+## showSocialLogin
+
+With social login enabled, a module can show Google or Facebook login buttons which the user can use to connect to their account. This will autofill the form elements with their account information such as name and email.
+
+| Value | Type | Behavior |
+|---|---|---|
+| false | boolean | `default` do not show social login buttons |
+| true | boolean | show social login buttons |  
+
+### Social Login /w Google & Facebook - [Live Preview](../../examples/preview/types/form/social.html)
```

Comment:
Feel free to throw away this feedback, but even though social login is only a part of the form type, `pathfora.integrateWithFacebook` and `pathfora.integrateWithGoogle` are top-level methods and ought to get their own section of the docs outside of the form configuration details.